src/components/OrbitCloseupText.tsx

```diff
@@ -53,6 +53,16 @@ const OrbitCloseupText: React.FC<OrbitCloseupTextProps> = ({
   const headerIndex = id + "_header";
   const subHeaderIndex = id + "_subheader";
   const bodyIndex = id + "_body";
+  const isMobileDevice = window.innerWidth <= 500;
+
+  // Calculate which close button to use
+  const closeButton = {
+    className: isMobileDevice
+      ? "orbit_closeup_back_button_mobile"
+      : "orbit_closeup_back_button",
+    src: isMobileDevice ? close_mobile : close,
+  };
+
```

Comment:
I used this in order to  use the same html img tag for both mobile and  regular sizes.
Previously there was one for each.